### DISCUSSION TITLE
MW4 Simulation Run

### DISCUSSION THREAD

**Discussion by BBRAUNCCP** [mworion/MountWizzard4#333] [Q&A] [answered]
Hi,
I'll be receiving my GM1000 in a few weeks & I'd like to practice a bit. I'm wondering if there's a way to use MW4 in a simulation mode? I've sen a video on YT that was labeled as a simulation run.  I've setup a profile with an ASCOM sim camera, Focuser, ASTAP, & Dome, but don't see a way to simulate the mount. 
Thanks,
Bruce

**Comment by mworion** [comment, marked as answer]
Hi Bruce,
unfortunately there is no simulation mode for the mount itself, because "simulating" the mount computer with all it's features can't be done right.
There are most of the other devices and functions of MW4 available also without mount connected (like environ data, seen, satellite etc), but some really require the mount to be connected.
Hope this helps
Michel

**Reply by BBRAUNCCP** [reply]
Hi Michel,

That’s perfectly understandable. I’ve been familiarizing myself with MW4 over the past few days & am very excited to be using it with my new mount when it arrives! Thanks very much for you labor creating such a useful & interesting program!
Best & CS,
Bruce
